### ISSUE TITLE
Personal Search preferences accessibility issues

### ISSUE THREAD

**Issue by stephanegigandet** [openfoodfacts/openfoodfacts-server#5818]
### Describe the bug

Some accessibility issues reported by the Accessibility Fundation on the personal search preferences on the website:

"On version 87 of Chrome I cannot reach the "Edit your food preferences" button with the keyboard and open it with the keyboard. This also applies to the button "close" & "Delete all preferences" if I want to close the preferences.

The radio buttons with the options "Not important, Important, etc." are not provided with a label (which falls under success criteria 1.3.1) so that the screen reader only reads the options and not for which the choice can be made. See example: () https://www.w3.org/WAI/tutorials/forms/grouping/) and (https://www.w3.org/WAI/WCAG21/Techniques/html/H71.html).

The logos of the companies are not provided with an alternative text in the alt = "" attribute. Not bad in itself because it is decorative, but it could help to improve the findability of your website in the search engines. If it doesn't interest you I would advise you to include the alt = "" attribute and leave it blank."

### To Reproduce

Go to https://world.openfoodfacts.org

Try navigating with the keyboard (tab + enter)  to select preferences.

### Expected behavior

Preferences can be set with the keyboard.

### Type of device

Browser
